_docs/tasks/traffic-management-v1alpha3/ingress.md

```diff
@@ -143,21 +143,43 @@ In our case, we have set the selector value to `istio: ingressgateway` to use th
 `istio-ingressgateway` controller. Therefore, to test our `Gateway` we will send requests to
 the `istio-ingressgateway` service.
 
-1. Get the ingressgateway controller pod's hostIP:
+1. Determine the ingressgateway URL:
+
+   * If your cluster is running in an environment that supports external load balancers, use the ingressgateway external address:
 
    ```command
-   $ kubectl -n istio-system get po -l istio=ingressgateway -o jsonpath='{.items[0].status.hostIP}'
-   169.47.243.100
+   $ kubectl -n istio-system get svc istio-ingressgateway
+   NAME                   TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)                                      AGE
+   istio-ingressgateway   LoadBalancer   172.21.232.178   130.211.10.121  80:31380/TCP,443:31390/TCP,31400:31400/TCP   1h
    ```
 
-1. Get the istio-ingress service's nodePorts for port 80 and 443:
-
    ```command
-   $ kubectl -n istio-system get svc istio-ingressgateway
-   NAME                   CLUSTER-IP     EXTERNAL-IP   PORT(S)                      AGE
-   istio-ingressgateway   10.10.10.155   <pending>     80:31486/TCP,443:32254/TCP   32m
+   $ export INGRESS_HOST=130.211.10.121
    ```
 
+   * If load balancers are not supported, use the ingressgateway controller pod's hostIP:
+
+        *  _GKE:_
```

Comment:
You don't need this: if you just deploy the ingressgateway service it will be assigned a public static IP on GKE (we ship config to make that service a LoadBalancer service in k8s by default).